src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -90,7 +84,7 @@ Yes, existing CockroachDB {{ site.data.products.cloud }} customers can pay for t
 
 ### Does the CockroachDB {{ site.data.products.cloud }} technical SLA cover clusters on Azure?
 
-During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), the [CockroachDB {{ site.data.products.cloud }} technical SLA](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) does not apply to CockroachDB {{ site.data.products.dedicated }} clusters on Azure. For more details about the roadmap, contact your Cockroach Labs account team.
```

Comment:
same comment as above, consider removing the whole question as its not unique to Azure.

Reply:
Done